eng/CodeGeneration.targets

```diff
@@ -5,18 +5,20 @@
     <_AutoRestCoreVersion>3.0.6257</_AutoRestCoreVersion>
     <_AutoRestCSharpVersion>https://github.com/Azure/autorest.csharp/releases/download/3.0.0-dev.20200323.2/autorest-csharp-v3-3.0.0-dev.20200323.2.tgz</_AutoRestCSharpVersion>
     <_SupportsCodeGeneration Condition="'$(IsClientLibrary)' == 'true'">true</_SupportsCodeGeneration>
-    <AutoRestConfiguration Condition="'$(AutoRestConfiguration)' == ''">autorest.md</AutoRestConfiguration>
-
+    <_DefaultInputName>$(MSBuildProjectDirectory)/autorest.md</_DefaultInputName>
```

Comment:
Would it better to just have the local autorest.md link to the remote config? Instead of passing it explicitly here?

Reply:
Autorest doesn't allow it, it has to have a single main input, this why I need to do this little dance.